crates/cli/src/lib.rs

```diff
@@ -504,7 +518,7 @@ pub fn test(matches: &ArgMatches, target: Target) -> io::Result<i32> {
         render: roc_reporting::report::RenderTarget::ColorTerminal,
         palette: roc_reporting::report::DEFAULT_PALETTE,
         threading,
-        exec_mode: ExecutionMode::Test,
+        exec_mode: exec_mode.clone(),
```

Comment:
```suggestion
        exec_mode,
```

This should fix the issue in CI.

Reply:
Thanks Ayaz :)